### PR TITLE
network, istio: Introduce pkg/network/istio package

### PR DESCRIPTION
**Pull request by rhrazdil**
Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

**What this PR does / why we need it**:
Moves Istio related constants and functions to separate
package pkg/network/istio

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by qinqon** [comment]
@rhrazdil can you fix conflict before review ?

File: pkg/network/istio/consts.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package consts
+package istio
```
Comment (qinqon) [inline pkg/network/istio/consts.go:1]: call this file annotations, the fact that there are const in side is not relevant and open the door to place here rnadom istio stuff.

**Review by qinqon — CHANGES_REQUESTED** [review]
Makes a lot of sense just one nite on const.go file name

File: pkg/network/istio/consts.go (hunk 1 above)
Comment (rhrazdil) [reply pkg/network/istio/consts.go]: Done along with the rebase, Thanks!

File: pkg/network/istio/annotations.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package consts
+package istio
```
Comment (EdDev) [inline pkg/network/istio/annotations.go:1]: Please add a license header.

File: pkg/network/istio/ports.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package istio
```
Comment (EdDev) [inline pkg/network/istio/ports.go:1]: Please add a license header

File: pkg/network/istio/proxy.go [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package istio
```
Comment (EdDev) [inline pkg/network/istio/proxy.go:1]: Please add the license header.

**Review by EdDev — CHANGES_REQUESTED** [review]
Some small nits, other than that it looks great. Thanks!

File: pkg/network/istio/annotations.go (hunk 2 above)
Comment (rhrazdil) [reply pkg/network/istio/annotations.go:20]: Thanks, I always forget that, Done

File: pkg/network/istio/ports.go (hunk 3 above)
Comment (rhrazdil) [reply pkg/network/istio/ports.go:20]: Done

File: pkg/network/istio/proxy.go (hunk 4 above)
Comment (rhrazdil) [reply pkg/network/istio/proxy.go:20]: Done

**Comment by rhrazdil** [comment]
Rebased

**Review by EdDev — APPROVED** [review]
Thank you!

**Pull request merged**